apps/server/src/modules/files-storage/files-storage.const.ts

```diff
@@ -0,0 +1,5 @@
+export enum FilesStorageInternalActions {
+	downloadBySecurityToken = '/file/internal/downloadBySecurityToken/:token',
+	updateSecurityStatus = '/file/internal/updateSecurityStatus/:token',
+}
+export const ApiVersionPath = '/api/v3';
```

Comment:
`ApiVersionPath` looks like a class name. Maybe we should use `API_VERSION_PATH` instead?